public/css/shorten_result.css

```diff
@@ -12,3 +12,14 @@
 .content-div {
     text-align: center;
 }
+
+.qrCodeContainer {
```

Comment:
Let's try to use consistent CSS selectors; dashes over camel case

Reply:
Done